src/main/java/org/folio/rest/impl/PurchaseOrderHelper.java

```diff
@@ -661,6 +669,45 @@ private void validateRenewalInfo(CompositePurchaseOrder compPO) {
       .thenApply(v -> compPO);
   }
 
+  private CompositePurchaseOrder validateFundDistributionTotal(CompositePurchaseOrder compPO) {
+    if (compPO.getWorkflowStatus() == WorkflowStatus.OPEN) {
```

Comment:
now this method is part of the `openOrder`, it guarantees that the order status is open